frontend/src/components/Util/TaskEditors/TaskEditor/index.tsx

```diff
@@ -99,11 +100,13 @@ function TaskEditor(
       onBlur();
     }
   };
-  const onSaveClicked = (): void => {
+  const onSave = (): boolean => {
```

Comment:
create a github issue for this after merging

Reply:
What is the issue? Are the functions not named correctly still?

Reply:
The linter warning